projects/vdk-plugins/vdk-test-utils/src/vdk/plugin/test_utils/util_plugins.py

```diff
@@ -57,7 +57,7 @@ def new_connection(self):
         import sqlite3
 
         print(self.__db_name)
-        return sqlite3.connect(f"{self.__db_file}")
+        return sqlite3.connect(f"{self.__db_file}", isolation_level=None)
 
```

Comment:
I am changing only vdk-core in this change. And any changes needed to make vdk-core .  

I am not sure why sqlite3 is started with autocommit False in 3.11 and while it's true in 3.10. But setting islation_level=None fixes it.

Reply:
Interesting, is that what you referred to https://docs.python.org/3/library/sqlite3.html ?

> isolation_level ([str](https://docs.python.org/3/library/stdtypes.html#str) | None) – The [isolation_level](https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.isolation_level) of the connection, controlling whether and how transactions are implicitly opened. Can be "DEFERRED" (default), "EXCLUSIVE" or "IMMEDIATE"; or None to disable opening transactions implicitly. See [Transaction control](https://docs.python.org/3/library/sqlite3.html#sqlite3-controlling-transactions) for more.

It doesn't seem true was an option.

Reply:
What's wrong with committing it / setting it to `DEFERRED` to keep backwards-compatible contract?

Reply:
Because I want to disable opening transactions at all. Every single statement is automatically committed.

Reply:
Okay...however wouldn't the VDK upgrading users then automatically transition from DEFERRED to None, so that suddenly no transactions for them from now on? Isn't that a breaking change, and in case  we decide to have such, should we add a disclaimer at least? Or our presumption based on current sqlite3  adoption stats show us this is a no-concern somehow